packages/salesforcedx-vscode-apex/package.json

```diff
@@ -388,12 +390,12 @@
       {
         "language": "apex",
         "scopeName": "source.apex",
-        "path": "./node_modules/@salesforce/apex-tmlanguage/grammars/apex.tmLanguage"
+        "path": "./grammars/apex.tmLanguage"
```

Comment:
With npm workspaces the grammar files are not found in the node_modules at the root of the monorepo.  To change this config to look there caused warning messages during local development.  This solution works for both vsix and local dev.